controllers/svc/env/env_cache_impl.go

```diff
@@ -2,13 +2,15 @@ package env
 
 import (
 	c "github.com/Apicurio/apicurio-registry-operator/controllers/common"
+	"github.com/kballard/go-shellquote"
```

Comment:
Is this necessary or is it an improvement?

Reply:
Improvement

Reply:
I would leave this out if possible just to avoid hitting new edge cases.
I'd encourage you to get this in `main` though.
Ideally this PR should be the smallest exact bug fix as possible.